File: client/Splendor Client/Assets/Scripts/ScriptableObjects/Scripts/LobbyPlayer.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.CompilerServices;
using UnityEngine;

[System.Serializable]
public class LobbyPlayer {
    public string username, access_token, refresh_token, expires_in;

    public LobbyPlayer() {}
    public LobbyPlayer(string username, string access_token, string refresh_token, string expires_in) {
        this.username = username;
        this.access_token = access_token;
        this.refresh_token = refresh_token;
        this.expires_in = expires_in;
    }

    public LobbyPlayer(Authentication authentication) {
        this.username = authentication.username;
        this.access_token = authentication.access_token;
        this.refresh_token = authentication.refresh_token;
        this.expires_in = authentication.expires_in;
    }
}

File: server/server/src/main/java/ca/mcgill/splendorserver/models/LobbyPlayer.java
package ca.mcgill.splendorserver.models;

public class LobbyPlayer {
  private String username;
  private String accessToken;
  private String refreshToken;
  private String expiresIn;

  public String getUsername() {
    return username;
  }

  public void setUsername(String username) {
    this.username = username;
  }

  public String getaccessToken() {
    return accessToken;
  }

  public void setaccessToken(String accessToken) {
    this.accessToken = accessToken;
  }

  public String getrefreshToken() {
    return refreshToken;
  }

  public void setrefreshToken(String refreshToken) {
    this.refreshToken = refreshToken;
  }

  public String getexpiresIn() {
    return expiresIn;
  }

  public void setexpiresIn(String expiresIn) {
    this.expiresIn = expiresIn;
  }
}
